src/kit.ts

```diff
@@ -53,6 +53,11 @@ export interface LoginResult {
     session: Session
 }
 
+export interface LogoutContext {
+    session: SerializedSession | Session
```

Comment:
Can we use only `Session` type here, protocol-scatter is using the `chain` of a session, it should be a `ChainDefinition` type, not a `chainId`, if i am not wrong, then `chain` in `SerializedSession` is only a `chainId`.
https://github.com/wharfkit/protocol-scatter/blob/master/src/index.ts#L24

Reply:
Done! Let me know if you run into any other issues!